h/static/scripts/directives/markdown.coffee

```diff
@@ -336,7 +336,7 @@ markdown = ['$filter', '$sanitize', '$sce', '$timeout', ($filter, $sanitize, $sc
     # React to the changes to the input
     inputEl.bind 'blur change keyup', ->
       ctrl.$setViewValue inputEl.val()
-      scope.$digest()
+#      scope.$digest()  # TODO: this shouldn't be a problem
```

Comment:
Ah, I see it, thanks @gergely-ujvari for the pointers... that is weird. I don't claim to be an angular expert but rewriting:
```
    inputEl.bind 'blur change keyup', ->
      ctrl.$setViewValue inputEl.val()
      scope.$digest()  # TODO: this shouldn't be a problem
```
To be this:
```
    inputEl.bind 'blur change keyup', ->
      $timeout-> ctrl.$setViewValue inputEl.val()
```

Solved the problem, and everything still works even with Math rendering, so that could be a possible solution.